tests/system/action/motion_category/test_number_motions.py

```diff
@@ -0,0 +1,437 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class MotionCategoryNumberMotionsTest(BaseActionTestCase):
+    def test_good_single_motion(self) -> None:
+        self.create_model(
+            "meeting/1",
+            {"name": "meeting_1", "motion_category_ids": [111], "motion_ids": [69]},
+        )
+        self.create_model(
+            "motion_category/111",
+            {
+                "name": "name_MKKAcYQu",
+                "prefix": "prefix_111",
```

Comment:
the prefix is a bit confusing. Please remove the numbers.

Reply:
Please do this for all categories in the test

Reply:
Renamed.